### PR TITLE
grafana/ui: Fix RelativeTimeRange supported formats

### LINKED ISSUE #36154: Datepicker on alert rules screen fails to accept some relative time range configurations
Hi, thank you for great work on grafana 8 :)

I found what seems to be a bug in the time picker on Create Alert Rule page. I want the time range to be "Yesterday", which according to [the docs](https://grafana.com/docs/grafana/latest/dashboards/time-range-controls/
) should be set as `now-1d/d` on both _From_ and _To_ fields. This configuration works in the dashboard time picker control, but not in the Create Alert Rule time picker, which gives me a "Value not in relative time format" error.

This is from the dashboard page, which works.
valid

This is from Create Alert Rule page, which doesn't work.
invalid

**Environment**:
- Grafana version: 8.0.4
- OS Grafana is installed on: debian stretch

Thanks for your time.

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:
The `RelativeTimeRangePicker` was linking to the [grafana docs](https://grafana.com/docs/grafana/latest/dashboards/time-range-controls/) which states that you can use a syntax like `now/w` for setting the time range to "this week so far". Alert rules (the only usage of this timepicker) does not support this notion. We save rules with a time range in absolute seconds (from now-10m to now would be from: 600s to: 0 for example).

Therefore  for now we'd like to remove the reference to the full docs and only support `now-[digit]s/m/h/d/w` until we potentially can adjust the alert rules to support the "x so far notion".

**Which issue(s) this PR fixes**:

Closes https://github.com/grafana/grafana/issues/36154

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx [hunk 1]
```diff
@@ -115,14 +116,22 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps): Re
...
-                      docs <Icon name="external-link-alt" />
-                    </a>
-                    .
-                  </div>
+                  <TimePickerTitle>
+                    <Tooltip
+                      content={
+                        <>
+                          <div>Supported formats</div>
+                          <code className={styles.tooltip}>now-[digit]s/m/h/d/w</code>
```
Comment (konrad147) [inline packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:124]: As we have a tooltip now, maybe we could add a few examples of the time ranges?

Comment (gillesdemey) [reply packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:124]: I think this is a good idea for users that might not be very familiar with the syntax

**Review by gillesdemey — APPROVED** [review]
LGTM! 👏

**Review by konrad147 — APPROVED** [review]
LGTM!

**Pull request merged**